### DISCUSSION TITLE
유니티 개발 월드 제작 시 문의

### DISCUSSION THREAD

**Discussion by ellejou76** [naverz/zepeto-studio-kor#609] [5. Q&A]
안녕하세요.  

유니티 3d 로 월드 제작 시  아래 내용이 가능한지 문의드립니다.
최대 용량 500 MB 만 문제가 없다면,

- 가이드에는 빌드잇 아이템 사용만 소개되고 있어서, 자체적으로 직접 제작한 오브젝트( 건물, NPC, 아이템 ) 유니티에서  사용 가능한가요?  

- 공식 툴로 마야를 사용 한다고 하는데.  의상 아이템 처럼,  블렌더 3D로 제작 해서 사용해도 기술적인 문제는 없을까요?

감사합니다.

**Comment by Zepeto-TechEvangelist** [comment]
안녕하세요~  

1. 자체적으로 직접 제작한 오브젝트도 사용 가능하며, 제페토의 가이드라인 기준에 해당하는 것이라면 에셋 스토어에서 가져온 것도 사용이 가능합니다.
컨텐츠 가이드라인을 참고하셔서 멋진 월드를 제작해주시기를 응원합니다.
https://github.com/naverz/zepeto-studio-kor/discussions/595

2. 블렌더 3D로 제작해서 사용해도 기술적인 문제는 없습니다.

감사합니다.
